<launch>
  <arg name="robot_name" default="beep" />
  <arg name="sim" default="false" />
  <arg name="standalone" default="true" />


  <param if="$(arg standalone)" name="robot_description" command="$(find xacro)/xacro --inorder '$(find poli2_description)/robots/$(arg robot_name).urdf.xacro'" />

  <group ns="tilt_motor">

    <node if="$(arg sim)" name="tilt_motor_sim" pkg="basic_sim" type="sim_node" output="screen">
      <rosparam command="load" file="$(find pan_tilt)/config/sim_tilt_driver.yaml" />
    </node>

    <rosparam command="load" file="$(find pan_tilt)/config/tilt_controllers.yaml" />
    <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
          output="screen" args="joint_state_controller position_controller">
    </node>
  </group>

</launch>

